test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Metadata/FormFileBindingSourceMetadataProviderTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Metadata
+{
+    public class FormFileBindingSourceMetadataProviderTests
+    {
+        [Fact]
+        public void ChecksParameterType_AssignsFormFileBindingSource()
```

Comment:
https://github.com/aspnet/Home/wiki/Engineering-guidelines#unit-test-method-naming

Consider something like `CreateBindingMetadata_ForMatchingType_SetsBindingSource`.

if you don't have a lot to say then say a little.